src/components/SearchAndFilter/FilterPanelSection/FilterPanelSection.js

```diff
@@ -23,50 +29,94 @@ const FilterPanelSection = ({ data, addToSelected, searchData }) => {
     setOverflowCounter(overflowCount);
   };
 
+  // Check if search term characters matches any characters in panel heading
+  const searchTermInHeading = hightlightSubString(heading, searchTerm).match;
+
+  // Serialise chip values into string so it can be interrogated with subString
+  let chipValues = [];
+  Object.entries(chips).forEach((chipValue) => {
+    chipValues.push(chipValue[1].value);
+  });
+  // Search chips for character match with search term
+  const searchTermInChips = hightlightSubString(
+    chipValues.toString(),
+    searchTerm
+  ).match;
```

Comment:
This seems a bit convoluted (`hightlightSubstring` has two jobs), why not `chipValues.includes(searchTerm)`?

Reply:
Because the search is fuzzy, array.includes must be an exact match to return true.